components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenManagementDAO.java

```diff
@@ -67,6 +67,16 @@ void updateAppAndRevokeTokensAndAuthzCodes(String consumerKey, Properties proper
                                                String[] authorizationCodes, String[] accessTokens)
             throws IdentityOAuth2Exception, IdentityApplicationManagementException;
 
+    default void revokeTokens(String consumerKey, String[] accessTokens)
```

Comment:
can we method comment please

Reply:
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2484/commits/4b81c64df3e3c71f8b1715cc02e6ad55e5d8ffe0